build-n-publish-release-w-docker.sh

```diff
@@ -41,28 +36,28 @@ pkgVersion=$versionPatch
 
 echo Publishing to Docker
 echo pkgName: $pkgName
-echo cdnBaseUrl: $cdnBaseUrl
-echo versionPatch: $versionPatch
-echo versionFull: $versionFull
-echo pkgVersion: $pkgVersion
-echo pkgBuild: $pkgBuild
-echo pkgCommitId: $pkgCommitId
+echo cdnBaseUrl: "$cdnBaseUrl"
+echo versionPatch: "$versionPatch"
+echo versionFull: "$versionFull"
+echo pkgVersion: "$pkgVersion"
+echo pkgBuild: "$pkgBuild"
+echo pkgCommitId: "$pkgCommitId"
 
 # TODO: Remove these steps in favor of publishing to `fromdoppler` organization.
 for environment in ${environments}; do
-    echo Publishing ${environment}...
+    echo Publishing "${environment}"...
```

Comment:
In my opinion in this way is better:

```suggestion
    echo Publishing "${environment}..."
```